app/services/hyrax/analytics/google.rb

```diff
@@ -30,7 +30,15 @@ def self.load_from_yaml
               Rails.logger.error("Unable to fetch any keys from #{filename}.")
               return new({})
             end
-            new yaml.fetch('analytics')&.fetch('google')
+            config = yaml.fetch('analytics')&.fetch('google', nil)
+            unless config
+              Rails.logger.info("Deprecated analytics format found. Please update your yaml file.")
```

Comment:
Elsewhere Hyrax is using `Deprecation.warn` instead of using the rails logger for deprecation messages.  I think it would be nice to give more guidance in the message.  Is there a wiki page or release notes that explain the expected configuration format?

Start of a suggestion:
```suggestion
              Deprecation.warn("Deprecated analytics configuration format found. Please update config/analytics.yml.")
```